src/java.base/share/classes/jdk/internal/classfile/impl/StackMapGenerator.java

```diff
@@ -429,7 +431,7 @@ private void processMethod() {
                     currentFrame.copyFrom(nextFrame);
                     nextFrame.dirty = false;
                 } else if (thisOffset < bcs.bci()) {
-                    throw new ClassFormatError(String.format("Bad stack map offset %d", thisOffset));
```

Comment:
@asotona Is this CFE an outdated implementation behavior from an older implementation? If yes, we can just use `generatorError("Bad stack map offset");` instead.